collectors/0/samza.py

```diff
@@ -101,5 +132,20 @@ def print_metric(metric_type, metric_name, ts, value, **tags):
         print ("%s.%s %d %s %s" % (metric_type.replace('org.apache.', ''), metric_name, ts, value, tags))
 
 
+def print_consumer_lag(ts, value, **tags):
+
+    if (unicode(str(value), 'utf-8').isnumeric()):
+        if tags:
+          tags = " " + " ".join("%s=%s" % (sanitize(name), v)
+                                for name, v in tags.iteritems())
+        else:
+            tags = ""
+        print ("%s %d %s %s" % (SAMZA_CONSUMER_LAG_METRIC_NAME, ts, value, tags))
```

Comment:
Well, the actual samza metrics from Kafka looks like this:

```
{
	'org.apache.samza.system.kafka.KafkaSystemConsumerMetrics': {
		'kafka-nogoalids-0-messages-behind-high-watermark': 0, 
		'kafka-nogoalids-1-messages-behind-high-watermark': 0,
		'kafka-nogoalids-2-messages-behind-high-watermark': 0,
		...
	}
}
```

I really don't want to name it `samza.system.kafka.KafkaSystemConsumerMetrics.kafka-nogoalids-messages-behind-high-watermark`. Maybe `samza.system.kafka.KafkaSystemConsumerMetrics.lag`?